hook-consumer/src/consumer.rs

```diff
@@ -42,7 +42,8 @@ impl<'p> WebhookConsumer<'p> {
         let client = reqwest::Client::builder()
             .default_headers(headers)
             .timeout(request_timeout)
-            .build()?;
+            .build()
+            .expect("failed to construct reqwest client for webhook consumer");
```

Comment:
There is nothing fallible in this method if we make this change (as we'll panic), so we can return `Self` instead of a `Result.

Reply:
Good eye! I feel like clippy or something should tell me that. 🤔